src/node_api/core/routes.rs

```diff
@@ -271,7 +275,9 @@ impl Client {
     pub async fn get_output(&self, output_id: &OutputId) -> Result<OutputResponse> {
         let path = &format!("api/v2/outputs/{}", output_id);
 
-        self.node_manager.get_request(path, None, self.get_timeout()).await
+        self.node_manager
+            .get_request(path, None, self.get_timeout(), true, true)
```

Comment:
Not sure if needed here for quorum anymore, because with the inputs commitment a transaction would be invalid then if it's wrong

Reply:
Right